src/test/java/browsertests/day2.java

```diff
@@ -0,0 +1,15 @@
+package browsertests;
+
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class day2 {
```

Comment:
Classes should start with capital letters and not include numbers.

Consider naming this to `DayTwo`